connect-examples/v2/node_payment/README.md

```diff
@@ -1,11 +1,9 @@
-== README
-
-Rename config.json.sample to config.json and fill in values for APP_ID & APP_TOKEN
+Rename config.json.sample to config.json and fill in values for `APP_ID` & `APP_TOKEN`
 
 To get the app running:
 
-* npm install
+* `npm install`
 
-* npm start
+* `npm start`
 
 * open a browser and navigate to [localhost:3000](localhost:3000)
```

Comment:
This link doesn't work (at least not in the rendered version of GitHub). Can you change it to: 
```[http://localhost:3000/](http://localhost:3000/)```